src/curated-corpus/components/ProspectListCard/ProspectListCard.tsx

```diff
@@ -68,12 +74,15 @@ export const ProspectListCard: React.FC<ProspectListCardProps> = (
                 : '/placeholders/collectionSmall.svg'
             }
             alt={prospect.title ?? 'No title supplied'}
-            className={classes.image}
+            sx={{
+              borderRadius: 4,
+              border: `1px solid ${curationPalette.lightGrey}`,
+            }}
           />
 
           <List dense disablePadding>
             <ListItem disableGutters>
-              <ListItemIcon className={classes.listItemIcon}>
+              <ListItemIcon sx={{ minWidth: '1.5rem' }}>
```

Comment:
Given that it's used in half a dozen places, would a styled component be worthwhile to add instead? There's already a `StyledListItemAction` on the feature branch, but that one specifies `minWidth` of `2rem`. Could we use that here? Could we update it to `1.5rem`, use it here, and make sure things don't fall apart visually on the Corpus/Schedule pages in the approved card component, too?

Reply:
Yep good call on re-using components. 

So far I don't really notice a difference when I change that value. I'm not even sure if `minWidth` is doing anything here but testing this out on the corpus page and the prospecting page has made me uncover some other style bugs. Should be quick fixes.

Reply:
I switched the `minWidth` to 1.5rem but didn't take effect. Seems like the style component style is being overriden by the default mui class? I tried to figure out why but will address this in a follow up ticket.

<img width="274" alt="Screenshot 2023-01-06 at 4 51 15 PM" src="https://user-images.githubusercontent.com/16694733/211106083-086a337c-0f68-4cea-b791-01009fcb9ffa.png">